src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.c

```diff
@@ -28,24 +29,141 @@ ActorInit Obj_Tokei_Tobira_InitVars = {
     (ActorFunc)ObjTokeiTobira_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80ABD750[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 300, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 300, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+Vec3f D_80ABD760 = { 0.0f, 0.0f, 80.0f };
+s16 D_80ABD76C[] = { -0x4000, 0x4000 };
+CollisionHeader* D_80ABD770[] = { &object_tokei_tobira_Colheader_0012B0, &object_tokei_tobira_Colheader_001590 };
+f32 D_80ABD778[] = { 1.0f, -1.0f };
+Gfx* D_80ABD780[] = { object_tokei_tobira_DL_001108, object_tokei_tobira_DL_0013E8 };
```

Comment:
```suggestion
s16 D_80ABD76C[] = {
    -0x4000, // OBJTOKEITOBIRA_TYPE_0
    0x4000,  // OBJTOKEITOBIRA_TYPE_1
};
CollisionHeader* D_80ABD770[] = {
    &object_tokei_tobira_Colheader_0012B0, // OBJTOKEITOBIRA_TYPE_0
    &object_tokei_tobira_Colheader_001590, // OBJTOKEITOBIRA_TYPE_1
};
f32 D_80ABD778[] = {
    1.0f,  // OBJTOKEITOBIRA_TYPE_0
    -1.0f, // OBJTOKEITOBIRA_TYPE_1
};
Gfx* D_80ABD780[] = {
    object_tokei_tobira_DL_001108, // OBJTOKEITOBIRA_TYPE_0
    object_tokei_tobira_DL_0013E8, // OBJTOKEITOBIRA_TYPE_1
};
```
All this data is indexed by type, so it's helpful to put comments on the data to visualize which type maps to which array element